samples/nrf5340/netboot/README.rst

```diff
@@ -68,11 +68,11 @@ To test the network core bootloader sample run the following commands:
 
 #. ``west flash -d build_netboot``
 
-   Flash the merged hex files for both domains (application core and network core).
+   Program the merged hex files for both domains (application core and network core).
```

Comment:
This might need an `--erase` or `--force`, but after next upmerge:
https://github.com/zephyrproject-rtos/zephyr/blob/master/scripts/west_commands/runners/nrfjprog.py#L195

@oyvindronningstad can you take a look at this description ?

Reply:
The comment is not directly related to the edit, so I suggest to update this in a different PR.